pipeline/inputs/windows-exporter-metrics.md

```diff
@@ -90,9 +95,89 @@ You can test the expose of the metrics by using _curl:_
 curl http://127.0.0.1:2021/metrics
 ```
 
+### Service where clause
+
+Windows service collector will retrieve entire service information in the local box.
```

Comment:
I haven't heard for managing Window service inside the dockerized Windows container before.
Surprisingly, it can be handled inside the environment...!

ref: https://stackoverflow.com/questions/60541158/dockerizing-a-windows-service

Reply:
I mean a container is just a "properly" isolated process so really this is relevant to whatever isolation people may apply :-)